### ISSUE TITLE
Notifications header not keyboard accessible

### ISSUE THREAD

**Issue by cleidigh** [Microsoft/vscode#45030]
The notifications header is not keyboard accessible and therefore not allowing access to 
the expand, collapse and clear all buttons.

![image](https://user-images.githubusercontent.com/25272315/36957473-a886b7e4-2002-11e8-895a-c3658fe30781.png)

**Comment by cleidigh** [comment]
@bpasero 
After further consideration this is probably as designed.
while you cannot focus the header there are global commands for both show notifications
and clear all notifications.  I think that's perfectly sufficient.

I'm going to go ahead and close this , reopen if you think this makes sense.

**Comment by bpasero** [comment]
@cleidigh they are eventually keyboard accessible when you Tab around often enough. But yeah, there are global keybindings to do the same.
